msteams-platform/graph-api/activity-feed/test-activity-feed-json-file.md

```diff
@@ -0,0 +1,706 @@
+								],
+								"type": "text/javascript"
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"type": "text",
+								"value": "application/x-www-form-urlencoded"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "grant_type",
+									"value": "password",
+									"type": "text"
+								},
+								{
+									"key": "username",
+									"value": "AlexW@M365x347208.OnMicrosoft.com",
+									"type": "text"
+								},
+								{
+									"key": "password",
+									"value": "07Apples!",
+									"type": "text"
+								},
+								{
+									"key": "client_id",
+									"value": "f97677f9-8607-476e-9759-8c45b2832b59",
+									"type": "text"
+								},
+								{
+									"key": "client_secret",
+									"value": "3RQa7TYXTU9ezMEu/O4x8pSNtkpM8tpW8m3foVjPVSc=",
+									"type": "text"
+								},
+								{
+									"key": "scope",
+									"value": "https://canary.graph.microsoft.com/.default",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "https://login.microsoftonline.com/60bcdb18-3e8b-4c8d-8e03-c9bd0a442378/oauth2/v2.0/token",
+							"protocol": "https",
+							"host": [
+								"login",
+								"microsoftonline",
+								"com"
+							],
+							"path": [
+								"60bcdb18-3e8b-4c8d-8e03-c9bd0a442378",
+								"oauth2",
+								"v2.0",
+								"token"
+							]
+						},
+						"description": "Authenticate with AAD"
+					},
+					"response": []
+				},
+				{
+					"name": "Send Custom Notification to User",
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"name": "Content-Type",
+								"value": "application/json",
+								"type": "text"
+							},
+							{
+								"key": "Authorization",
+								"value": "Bearer {{graph_access_token}}",
+								"type": "text"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": "{\r\n    \"about\": \"https://graph.microsoft.com/beta/teams/e5a548cb-e5a4-4e54-8da1-ec0e944d9c11\",\r\n    \"activityType\": \"taskCreated\",\r\n    \"previewText\": \"New Task Created\",\r\n    \"onClickWebUrl\": \"https://teams.microsoft.com/l/channel/19%3Abdbb89ee4acb46ee9f704057c12163a3%40thread.skype/General?groupId=e5a548cb-e5a4-4e54-8da1-ec0e944d9c11\",\r\n    \"recipient\": {\r\n        \"@odata.type\": \"microsoft.graph.aadUserNotificationAudience\",\r\n        \"userId\": \"2725adbe-59ac-4b1a-8096-b47bd244eb09\"\r\n    },\r\n    \"templateParameters\": [\r\n    \t{\r\n\t\t\t\"name\": \"taskId\",\r\n\t\t\t\"value\": \"Task 12321\"\r\n\t\t}\r\n    ],\r\n    \"teamsAppId\": \"a1c8b1ca-3270-4007-9699-a8956baf73e8\"\r\n}\r\n"
+						},
+						"url": {
+							"raw": "https://canary.graph.microsoft.com/testprodbetatestTeamsGraphSvcDev/teamwork/generateActivityNotification",
```

Comment:
This doesn't look right. This is an older endpoint we should not be sharing. Can you double-check that the Postman collection you're using has 'sendActivityNotification' and is following the correct endpoints in the main doc? please ping me if you need more clarification.